src/components/Input.jsx

```diff
@@ -0,0 +1,52 @@
+const Variant = {
+  DEFAULT: 'DEFAULT',
+  SECONDARY: 'SECONDARY',
+  SMALL: 'SMALL',
+};
+
+const Input = ({
+  label,
```

Comment:
Lets add prop validations too to make sure that `label`, `name` and `placeholder` are always required and also define the proptypes for other attributes.

Reply:
Will do the changes.